chore: update variables (#9)
bytebase/terraform-provider-bytebase: README.md

cd terraform-provider-bytebase && make install

> # initial the terraform for your example

Reviewer: ```suggestion
# initialize the terraform for your example
```

> # you may need to change the service_account and service_key

Reviewer: ```suggestion
# you need to set the service_account and service_key to your own
```
